DataFormats/L1TMuon/interface/EMTFOutput.h

```diff
@@ -70,7 +71,8 @@ namespace l1t {
     emtf::EventTrailer GetEventTrailer()    { return EventTrailer;  };
     emtf::MTF7Trailer GetMTF7Trailer()      { return MTF7Trailer;   };
     emtf::AMC13Trailer GetAMC13Trailer()    { return AMC13Trailer;  };
-    const uint64_t Dataword() const     { return dataword; };
+    const int Format_Errors() const { return format_errors; };
+    const uint64_t Dataword() const { return dataword; };
```

Comment:
Why do you even have separate branches for the code for 8.0.x and 8.1.x ?

At the very least, they list of commits should be the same, cherry-picked form one release (whatever you consider the one people should be developing with) to the other.